internal/gatewayapi/testdata/gateway-with-listener-with-udproute-with-mismatch-port-protocol.out.yaml

```diff
@@ -100,3 +100,7 @@ xdsIR:
     - address: 0.0.0.0
       name: envoy-gateway/gateway-1/udp
       port: 10162
+      route:
+        destination:
+          name: udproute/default/udproute-1/rule/-1
+        name: udproute/default/udproute-1
```

Comment:
For HTTPRoute, we generate a route with a 500 direct response for routes with failed backends.

Reply:
Add a test in the xDS - it'll generate a cluster without endpoints.